### PR TITLE
Implement Access token modal like Link social modal

### LINKED ISSUE #4542: Implement Access token modal like Link social modal
link_social_modal

This is social media linking modal which is integrated using simple HTML - https://github.com/publiclab/plots2/blob/master/app/views/users/profile.html.erb#L206-L223 

access_token_modal
This is access token modal which is integrated using JavaScript - https://github.com/publiclab/plots2/blob/master/app/views/users/_token.html.erb which is complicated as token modal can be implemented similar to the social modal.

Make the token modal appear without using the JavaScript code present in file _token.html.erb and implement it just like social media modal i.e., normal bootstrap modal called on button press.

Thanks!

### PR DESCRIPTION
**Pull request by trompj**
Fixes #4542   (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by trompj** [comment]
Is this a successful contribution? One of my first ones after all!

**Comment by trompj** [comment]
image
Initial Code - JavaScript Only #4542 

screen shot 2019-01-07 at 10 55 08 pm
After HTML Streamline

screen shot 2019-01-07 at 10 58 07 pm
Compared with Social Media Modal

As you can see, both modals are similar, although they need some additional css work, which I am happy to look into. I also updated the text for Link Social media to Link Social Media. Seemed to flow better.

**Comment by shubhscoder** [comment]
@trompj great work :+1:  and congratulations for opening your first PR :tada: , I know how good it feels to open your first PR. Just one suggestion, add the issue number this PR fixes here. 
> Fixes #0000 (<=== Add issue number here)

Also, tick the appropriate checkboxes ! Thank you so much!

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -195,20 +195,39 @@
...
+    <a class="btn btn-default btn-block" href="/feed/<%= @user.name %>.rss"><i class="fa fa-rss" style="color:orange;"></i> <%= t('users.profile.rss_feed_for') %> <%= @user.name %></a>
     <% if current_user && current_user.uid == @user.uid %>
-      <a href="/profile/<%= @user.name %>/edit" class="btn btn-default btn-block"><i class="fa fa-pencil"></i> <%= t('users.profile.edit_profile') %></a> 
+      <a href="/profile/<%= @user.name %>/edit" class="btn btn-default btn-block"><i class="fa fa-pencil"></i> <%= t('users.profile.edit_profile') %></a>
     <% end %>
   </p>
 
   <% if @user && current_user && @user == current_user %>
-    <button id="btn-token" class="btn btn-default btn-block">My Access Token</button>
-    <button data-toggle="modal" data-target="#socialModal" class="btn btn-default btn-block">Link Social media</button>
```
Comment (gauravano) [inline app/views/users/profile.html.erb]: Please maintain the order of button like this only and shift the modal code below both the buttons.

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -310,6 +348,8 @@
     })();
 </script>
 
+
```
Comment (gauravano) [inline app/views/users/profile.html.erb]: Please remove this space.

**Comment by gauravano** [comment]
Also, please see the checklist and do the changes to PR accordingly as asked by @shubhscoder. Thanks!

**Comment by trompj** [comment]
Sounds great! Thanks for all the feedback, I will update the pull request over the next couple days.

**Comment by gauravano** [comment]
Issue number should be #4542. I am correcting it for now. Also, @trompj you have to make sure that no diff is recorded for `yarn.lock`. One possible way is copy the `yarn.lock` file from master branch and copy in your local setup and push it, so no diff is recorded. Also, there are other different methods too using Git, which you can search on stackoverflow or google. 

Thanks!

**Comment by trompj** [comment]
Alright @gauravano! I got it back to normal and it is completely removed from the file changes. My apologies for the misunderstanding there. Thanks for all of the help though. It's been really great.

**Comment by gauravano** [comment]
Merged :tada: :balloon: 

Hi @trompj, thanks for nice work. We would love to have more help from you. If you are interested in solving more issues then check here - https://code.publiclab.org/#r=all for a new issue. In case, you are unable to find an issue, feel free to comment here. Thanks!

**Comment by gauravano** [comment]
Awesome!

**Pull request merged**